lib/policy_actions/email_alert_signup_content_item_publisher.rb

```diff
@@ -0,0 +1,26 @@
+# Publishes a policy content item to the Publishing API
+class EmailAlertSignupContentItemPublisher
+  attr_reader :policy
+
+  def initialize(policy)
+    @policy = policy
+  end
+
+  def run!
+    publishing_api.put_content_item(base_path, content_item_payload)
+  end
+
+  def content_item_payload
+    EmailAlertSignupContentItemPresenter.new(policy, 'major').exportable_attributes
+  end
+
+private
+
+  def base_path
+    "#{policy.base_path}/email-signup"
```

Comment:
How about making the `base_path` method public on the presenter and then simply calling `content_item_payload.base_path`? That would be a nicer encapsulation as the way that an email signup path is generated will only exist in one place and not two.